pallets/ddc-clusters/src/weights.rs

```diff
@@ -31,108 +31,168 @@ pub trait WeightInfo {
+		Weight::from_parts(3_972_000_000_u64, 0)
+			.saturating_add(T::DbWeight::get().reads(15_u64))
+			.saturating_add(T::DbWeight::get().writes(6_u64))
 	}
 	// Storage: DdcClusters Clusters (r:1 w:0)
+	// Proof Skipped: DdcClusters Clusters (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcNodes StorageNodes (r:1 w:1)
-	// Storage: DdcClusters ClustersNodes (r:0 w:1)
+	// Proof Skipped: DdcNodes StorageNodes (max_values: None, max_size: None, mode: Measured)
+	// Storage: DdcClusters ClustersNodes (r:1 w:1)
+	// Proof Skipped: DdcClusters ClustersNodes (max_values: None, max_size: None, mode: Measured)
+	// Storage: DdcClusters ClustersNodesStats (r:1 w:1)
+	// Proof Skipped: DdcClusters ClustersNodesStats (max_values: None, max_size: None, mode: Measured)
 	fn remove_node() -> Weight {
-		Weight::from_parts(24_000_000_u64, 0)
-			.saturating_add(T::DbWeight::get().reads(2_u64))
-			.saturating_add(T::DbWeight::get().writes(2_u64))
+		Weight::from_parts(505_000_000_u64, 0)
+			.saturating_add(T::DbWeight::get().reads(4_u64))
+			.saturating_add(T::DbWeight::get().writes(3_u64))
 	}
 	// Storage: DdcClusters Clusters (r:1 w:1)
+	// Proof Skipped: DdcClusters Clusters (max_values: None, max_size: None, mode: Measured)
 	fn set_cluster_params() -> Weight {
-		Weight::from_parts(16_000_000_u64, 0)
+		Weight::from_parts(244_000_000_u64, 0)
 			.saturating_add(T::DbWeight::get().reads(1_u64))
 			.saturating_add(T::DbWeight::get().writes(1_u64))
 	}
 	// Storage: DdcClusters Clusters (r:1 w:0)
-	// Storage: DdcClusters ClustersGovParams (r:0 w:1)
-	fn set_cluster_gov_params() -> Weight {
-		Weight::from_parts(17_000_000_u64, 0)
-			.saturating_add(T::DbWeight::get().reads(1_u64))
-			.saturating_add(T::DbWeight::get().writes(1_u64))
+	// Proof Skipped: DdcClusters Clusters (max_values: None, max_size: None, mode: Measured)
+	// Storage: DdcClusters ClustersNodesStats (r:1 w:1)
+	// Proof Skipped: DdcClusters ClustersNodesStats (max_values: None, max_size: None, mode: Measured)
+	// Storage: DdcClusters ClustersNodes (r:1 w:1)
+	// Proof Skipped: DdcClusters ClustersNodes (max_values: None, max_size: None, mode: Measured)
+	fn validate_node() -> Weight {
+		Weight::from_parts(401_000_000_u64, 0)
+			.saturating_add(T::DbWeight::get().reads(3_u64))
+			.saturating_add(T::DbWeight::get().writes(2_u64))
 	}
 }
 
 // For backwards compatibility and tests
 impl WeightInfo for () {
 	// Storage: DdcClusters Clusters (r:1 w:1)
+	// Proof Skipped: DdcClusters Clusters (max_values: None, max_size: None, mode: Measured)
+	// Storage: DdcClusters ClustersNodesStats (r:0 w:1)
+	// Proof Skipped: DdcClusters ClustersNodesStats (max_values: None, max_size: None, mode: Measured)
 	// Storage: DdcClusters ClustersGovParams (r:0 w:1)
+	// Proof Skipped: DdcClusters ClustersGovParams (max_values: None, max_size: None, mode: Measured)
 	fn create_cluster() -> Weight {
-		Weight::from_parts(15_000_000_u64, 0)
+		Weight::from_parts(350_000_000_u64, 0)
```

Comment:
The weights were taken from Rakan's PR and was executed on a dedicated benchmarking machine